pretix_eth/txn_check.py

```diff
@@ -0,0 +1,75 @@
+import codecs
+import requests
+
+def json_rpc_request(method, params):
+    try:
+        payload = { "id": 1337, "jsonrpc": "2.0", "method": method, "params": params }
+        result = requests.post('https://api.myetherwallet.com/eth', data=payload)
+        json_result = result.json()
+        return json_result['result']
+    except NameError:
+        return None
+    except TypeError:
+        return None
+    except AttributeError:
+        return None
+    return None
+
+def check_txn_confirmation(txn_hash, from_address, to_address, currency, amount, timestamp):
+    from_address = from_addres.lower()
```

Comment:
Check this failing CI run: https://circleci.com/gh/esPass/pretix-eth-payment-plugin/22

This file currently has a number of simple errors like mispelled variable names.